client/src/components/MileageModal.vue

```diff
@@ -0,0 +1,175 @@
+<template>
+  <v-dialog v-model="value" :fullscreen="isFullscreen" width="500px">
+    <v-card height="600px" class="bg-backgroundGrey">
+      <div style="height: 10px">
+        <v-img src="/images/Footer-min.jpeg" width="100%" cover />
+      </div>
+      <v-spacer />
+      <v-form v-model="form">
+        <v-container class="px-8">
+          <v-row align="center">
+            <v-col>
+              <v-card-title class="mx-auto text-h5 justify-center">Add Mileage</v-card-title>
+            </v-col>
+          </v-row>
+          <v-row>
+            <v-card-text>
+              <v-row>
+                <v-col>
+                  <v-text-field
+                    type="date"
+                    label="Date"
+                    v-model="date"
+                    bg-color="white"
+                    :rules="[required]"
+                  />
+                </v-col>
+              </v-row>
+              <v-row>
+                <v-col>
+                  <p class="px-2 text-subtitle-2">Distance</p>
+                </v-col>
+              </v-row>
+              <v-row>
+                <v-col>
+                  <div v-if="tempIconType === 'running'" class="ma-0 pa-0" id="runner">
+                    <v-slider
+                      v-model="distance"
+                      color="green"
+                      :thumb-size="40"
+                      elevation="0"
+                      :step="1"
+                      min="1"
+                      max="100"
+                    />
+                  </div>
+                  <div v-if="tempIconType === 'walking'" id="walker" class="ma-0 pa-0">
+                    <v-slider
+                      v-model="distance"
+                      color="green"
+                      :thumb-size="40"
+                      elevation="0"
+                      :step="1"
+                      min="1"
+                      max="100"
+                    />
+                  </div>
+                  <div v-if="tempIconType === 'wheeling'" id="wheeler" class="ma-0 pa-0">
+                    <v-slider
+                      v-model="distance"
+                      color="green"
+                      :thumb-size="40"
+                      elevation="0"
+                      :step="1"
+                      min="1"
+                      max="100"
+                    />
+                  </div>
+                </v-col>
+              </v-row>
+              <v-row>
+                <v-col align="center">
+                  <v-chip class="px-6 text-h5 rounded" color="green">{{ distance }}</v-chip>
+                  <p class="text-subtitle-2">KILOMETERS</p>
+                </v-col>
+              </v-row>
+            </v-card-text>
+          </v-row>
+        </v-container>
+        <v-card-actions class="d-flex justify-center">
+          <v-btn @click="$emit('update:modelValue', !modelValue)">CANCEL</v-btn>
```

Comment:
I would make this the X icon on the top right so that it matches our other modals